### DISCUSSION TITLE
Trouble lualine component

### DISCUSSION THREAD

**Discussion by dpetka2001** [LazyVim/LazyVim#2875] [General]
Hi @folke. I'm trying out the new Troublev3 Extra and was wondering if one chooses to use `lsp_document_symbols` instead of `symbols` is there a way to specify the number of symbols to be rendered (something like the setting `depth` of `aerial`)?

**Comment by folke** [comment]
You can set `max_items` like the below. however, there was an issue with this specifically for the statusline compoinent, but I just fixed that, so make sure to update :)

```lua
      local symbols = trouble.statusline({
        mode = "lsp_document_symbols",
        groups = {},
        max_items = 3,
        title = false,
        filter = { range = true },
        format = "{kind_icon}{symbol.name:Normal}",
      })
```

**Reply by dpetka2001** [reply]
Works like a charm!! Appreciate it :smile:
